exonum/src/api/mod.rs

```diff
@@ -65,6 +65,10 @@ pub enum ApiError {
     #[fail(display = "Bad request: {}", _0)]
     BadRequest(String),
 
+    /// Not found.
```

Comment:
What is not found?

Reply:
? :)
https://en.wikipedia.org/wiki/HTTP_404